src/common/components/ErrorBoundary/index.tsx

```diff
@@ -5,15 +5,15 @@ type Props = {
 };
 
 type State = {
-  hasError: boolean;
+  readonly hasError: boolean;
 };
 
 export default class ErrorBoundary extends Component<Props, State> {
   static getDerivedStateFromError() {
     return { hasError: true };
   }
 
-  state = {
+  readonly state = {
```

Comment:
co toto?

    this.state.hasError = false;
v renderi vobec nehlasi chybu

ak spravim
state: State = { ... }

tak hlasi

Reply:
aha nieje to deep.

`this.state = {foo: 'bar'}` uz hlasi chybu

Reply:
```
 this.state = {hasError: true};
```
hlasi chybu len ak mam obe

```
  readonly state: State = {
    hasError: false,
  };
```

Reply:
WE WANT BOTH STATE INSTANCE PROPERTY AND IT'S KEYS TO BE READONLY.

FIX IT PLEASE.